KoansLib/KoanPrinter.php

```diff
@@ -15,12 +15,12 @@ class KoanPrinter extends CliTestDoxPrinter
 {
     public function printResult(TestResult $result): void
     {
-        $this->printHeader();
+        $this->printHeader($result);
```

Comment:
As it is right now, it doesn't.
Sincerely, I don't know what the result contains. @IvanSolaDelgado try printing the $result also on the printHeader method and let's see if it looks better.
If it makes sense, maybe we could create a new method named printBody($result) and place it between the header and the footer?

Reply:
If I print $result, a lot of information comes out of the entire execution, Asier and I have talked and we have thought that in the header we could think of a custom header for us (instead of writing two line breaks which is what it does right now), but for now we can leave it that way.

Reply:
ok!